inc/upgrades/namespace.php

```diff
@@ -129,7 +129,7 @@ function is_version_latest() : bool {
 	}
 
 	$releases = get_supported_version_info();
-	return $version === ( $releases[0]['version'] ?? $version );
+	return ( $version >= $releases[0]['version'] ?? false );
```

Comment:
Would this be preferred?
`return ( isset( $releases[0]['version'] ) && $version >= $releases[0]['version'] ?? false );`

Reply:
`??` is the [null coalescing operator](https://www.php.net/manual/en/migration70.new-features.php#migration70.new-features.null-coalesce-op) which replaces the need to use isset.

That said, the intention here is to handle if there's no value available to force the check to return true, so your code needs to be
```suggestion
	return $version >= ( $releases[0]['version'] ?? $version );
```